client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java

```diff
@@ -93,19 +92,28 @@
     @UiField(provided = true)
     BrandPicker brandPicker;
     @UiField
-    LIElement smallAnchor;
+    LIElement smallLI;
     @UiField
-    LIElement mediumAnchor;
+    LIElement mediumLI;
     @UiField
-    LIElement largeAnchor;
+    LIElement largeLI;
     @UiField
-    LIElement xlargeAnchor;
+    LIElement xlargeLI;
+    @UiField
+    AnchorElement smallAnchor;
```

Comment:
idem here. It's a link? Why not smallLink or something else? I would suggest not putting the type of the object in the name, but the meaning instead.